enigma/src/test/java/cuchaz/enigma/translation/mapping/serde/recaf/TestRecaf.java

```diff
@@ -26,21 +24,18 @@ public void testIntegrity() throws Exception {
             contents = Sets.newHashSet(new String(in.readAllBytes(), StandardCharsets.UTF_8).split("\\R"));
         }
 
-        try (FileSystem fs = Jimfs.newFileSystem()) {
+        Path path = Path.of("src/test/resources/recaf.mappings");
```

Comment:
This shouldn't be writing to the input file, it probably should be writing to a temp one

Reply:
Now it is not writes to input